charts/community-operator/templates/mongodbcommunity_cr_with_tls.yaml

```diff
@@ -51,7 +51,61 @@ spec:
   dnsNames:
     - "*.{{ .Values.resource.name }}-svc.{{ .Values.namespace }}.svc.cluster.local"
 {{- end }}
-
+{{- if .Values.resource.tls.useX509 }}
```

Comment:
I am no helm guru, but It seems that the previous options combined `.Values.resource.tls.useCertManager` as well. Since we are using it here as well,  we should do the same.